apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/manifest.json

```diff
@@ -173,6 +173,10 @@
         {
           "title": "Go-Live Checklist",
           "path": "go-live-checklist"
+        },
+        {

```

Comment:
I didn't notice this earlier, but I think we should move this recipe to Pre-Development under the `Developer Experience` section. This information needs to be thought about before the go-live status of a project.